dotnet/src/Extensions/Planning/Planning.Core.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Planning</AssemblyName>
```

Comment:
I thought the consolidated assembly was going to be called `Microsoft.SemanticKernel.Planners` and this would be the root namespace also.

@lemillermicrosoft Do you know what the decision is?

Reply:
That's what is current in #2668 but since that isn't finalized my assumption is those types of changes would happen then. I don't care either way, however.

Reply:
Forgot to update the name here - good catch!

I will update the name of the assembly and project here and everywhere once we have a consensus (didn't see the ADR until now).